packages/go/scenepack/scenepack_test.go

```
@@ -0,0 +1,655 @@
+			t.Fatalf("写 %s 失败: %v", rel, err)
+		}
+	}
+}
+
+func appendKnowledge(line string) func(*testing.T, string) {
+	return func(t *testing.T, dir string) {
+		f, err := os.OpenFile(filepath.Join(dir, "knowledge/bedtime_routine.md"), os.O_APPEND|os.O_WRONLY, 0o644)
+		if err != nil {
+			t.Fatalf("打开知识文件失败: %v", err)
+		}
+		defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)